client/src/Components/HeadStyles.jsx

```diff
@@ -0,0 +1,50 @@
+import styled from 'styled-components'
+<style>
+  @import url('https://fonts.googleapis.com/css2?family=ABeeZee:ital@0;1&display=swap');
```

Comment:
download font to assetes and load from there, sometimes we will not have good connection, and this will take time